coulomb-tests/jvm/src/test/scala/coulomb/SerializationTest.scala

```diff
@@ -0,0 +1,35 @@
+package coulomb
+
+import spire.math.{ Rational, Real, Algebraic }
+import spire.algebra._
+// pick up the various group/semigroup typeclasses
+import spire.std.any._
+
+import singleton.ops._
+
+import utest._
+
+import coulomb.si._
+import coulomb.temp._
+
+import coulomb.validators.CoulombValidators._
+
+object SerializationTests extends TestSuite {
+  val tests = Tests {
+    test("quantity be serializable") {
```

Comment:
should one of these be the Quantity serialize test

Reply:
Do you refer to the class name or the name of the test? One of the comes form QuantityTest, the other form TemperatureTest

Reply:
they seem to contain the same code, both testing Temperature

Reply:
Sorry my mistake while moving the tests, Fixed now

Reply:
It still looks the same, did you push the change?

Reply:
looking for quantity ser/de test:
```scala
    test("quantity be serializable") {
      import coulomb.scalatest.serde._
      val qs = Quantity[Int, Meter %/ Second](10)
      val qd = roundTripSerDe(qs)
      assert(qd.isValidQ[Int, Meter %/ Second](10))
      assert(qd === qs)
    }
```

Reply:
Rebased onto develop and now it includes the correct test